pkg/build/admission/defaults/admission.go

```diff
@@ -83,6 +83,15 @@ func (a *buildDefaults) applyBuildDefaults(build *buildapi.Build) {
 		addDefaultEnvVar(envVar, buildEnv)
 	}
 
+	sourceDefaults := a.defaultsConfig.SourceStrategyDefaults
+	sourceStrategy := build.Spec.Strategy.SourceStrategy
+	if sourceDefaults != nil && sourceDefaults.Incremental != nil && *sourceDefaults.Incremental &&
```

Comment:
essentially this feature is allowing you to change the defaulting behavior.  today it always defaults to false, w/ this feature you can force it to default to true.